### ISSUE TITLE
Fresh installation fails the health checks

### ISSUE THREAD

**Issue by ma-sadeghi** [jepegit/cellpy#276]
The following was done on a new installation from PyPI on a fresh condo virtualenv:

```shell
❯ cellpy info --check
=================================== checking ===================================
[cellpy] * - Checking cellpy imports
[cellpy] -> succeeded!
--------------------------------------------------------------------------------
[cellpy] * - Checking importing pyodbc
 reading prms
 - ODBC: pyodbc
 - SEARCH_FOR_ODBC_DRIVERS: True
 - use_subprocess: False
 - detect_subprocess_need: False
 - stated office version: 64bit
 checking system
 - running on posix
 - running on a mac
 - python version: 64bit
 - os version: 
 checking existence of mdb-export
 - failed finding it
 looks like you are on a mac (driver set to
 /usr/local/lib/libmdbodbc.dylib)
 - but cannot find it!
f[cellpy] -> failed!!!!
--------------------------------------------------------------------------------
[cellpy] * - Checking configuration (prm) file
[cellpy] ->/Users/amin/.cellpy_prms_amin.conf
[cellpy] File does not exist!
[cellpy] check raised an exception (expected str, bytes or os.PathLike object, not NoneType)
[cellpy] OH NO!!! You (or I) failed!
[cellpy] Failed 1 out of 3 checks.
================================================================================
```

- OS: macOS 11.7
- CPU: Intel

**Comment by ma-sadeghi** [comment]
This could be a macOS-related issue, since I was able to install it on my Ubuntu machine. I also noticed that your CIs only run on Ubuntu and Windows.

**Comment by jepegit** [comment]
Thanks! Yes, I found it a bit tricky to set up CIs, and in particularly for macOS. But its been some years since I last tried it, so well worth to try once more. Will do (see you opened another issue for that, so will use that for making a branch). Must admit that I also find it a bit fiddly to make tests for the cli tool and in particular things that are related to installation and setup. But from your print-out I see that the cellpy output is not very easy to understand (even I had to think a while, and I wrote it). (I think) the message tries to communicate that the computer is missing a library that it needs for parsing raw-data files from Arbin testers (old versions, .res format). I will double check and improve the feedback from cellpy.
